spki/src/traits.rs

```diff
@@ -173,3 +174,12 @@ where
         })
     }
 }
+
+/// Returns the `BitString` encoding of the signature.
+///
+/// X.509 and CSR structures require signatures to be BitString encoded.
+#[cfg(feature = "alloc")]
+pub trait SignatureBitStringEncoding: SignatureEncoding {
```

Comment:
I'm not sure I see the point in a mandatory `SignatureEncoding` supertrait bound unless `to_bitstring` is a provided method.

Reply:
yeah, that's a fair point.